src/pudl/output/ferc1.py

```diff
@@ -2274,36 +2274,45 @@ def annotated_forest(self: Self) -> nx.DiGraph:
         annotated_forest = deepcopy(self.forest)
         nx.set_node_attributes(annotated_forest, self.node_attrs)
         nx.set_edge_attributes(annotated_forest, self.edge_attrs)
-        annotated_forest = self.propagate_node_attributes(annotated_forest)
 
         logger.info("Checking whether any pruned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.pruned)
         logger.info("Checking whether any orphaned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.orphans)
-        self.check_conflicting_tags(annotated_forest)
+
+        annotated_forest = self.propagate_node_attributes(annotated_forest)
```

Comment:
i moved the propagation of tags to be after the checking of lost tags because that check doesn't make sense after the tags have been propagated. i added a note in the `check_lost_tags` method about why.